### DISCUSSION TITLE
Diary-11-我需要治疗！ | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#320] [Announcements]
# Diary-11-我需要治疗！ | Zi-Zi's Journey

2024.2.26-2024.03.10

https://gz-metal-cell.github.io/2024/03/10/Diary-11-%E6%88%91%E9%9C%80%E8%A6%81%E6%B2%BB%E7%96%97%EF%BC%81/

**Comment by promefire** [comment]
谢谢关哥给我治脱发/(ㄒoㄒ)/~~
